lib/bundler_api/cdn.rb

```diff
@@ -7,6 +7,7 @@ def self.purge_specs(client = self.client)
     client.purge_path '/latest_specs.4.8.gz'
     client.purge_path '/specs.4.8.gz'
     client.purge_path '/prerelease_specs.4.8.gz'
+    client.purge_path "/names"
```

Comment:
Yeah, right, I went sleep last night thinking on that. Will be updating this at any minute now.